addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/CommandInjectionPlugin.java

```diff
@@ -134,6 +136,77 @@
         // Used for *nix
         // OS_PAYLOADS.put("\"|\"ld", null);
         // OS_PAYLOADS.put("'|'ld", null);
+
+        /**
+         * Null Byte Payloads.
+         *
+         * <p>Say NIX OS is there,
+         *
+         * <p>original input: "image"
+         *
+         * <p>Vulnerable application is executing command as "cat" + <input value>+ ".jpeg".
+         *
+         * <p>ZAP payload: ";cat /etc/passwd"
+         *
+         * <p>final value passed to Vulnerable application is "image;cat /etc/passwd".
+         *
+         * <p>Command executed by application "cat image;cat /etc/passwd.jpeg" and
+         *
+         * <p>it will not succeed but if we add null byte to the payload then command executed by
+         * application: "cat image;cat /etc/passwd\0.jpeg" and hence .jpeg will be ignored and
+         * attack will succeed.
+         *
+         * <p>Here we are not adding null byte before the payload ie "image\0;cat /etc/passwd"
```

Comment:
`ie` → `i.e.` (there are other occurrences)

Reply:
sure.

Reply:
The comment in `TestRemoteFileInclude` needs this tweak.

Reply:
done. thanks